test/upgrade/prober/configuration.go

```diff
@@ -103,6 +105,7 @@ func NewConfig(namespace string) *Config {
 		},
 		Wathola: Wathola{
 			ConfigMap: ConfigMap{
+				ConfigTemplate:   ("../" + defaultConfigFilename),
```

Comment:
fmt.Sprintf("../%s", defaultConfigFilename),

is more readable and consistent with line 110

Reply:
this is changed.